Backpack/Calendar/Classes/BPKCalendarPriceLabelCell.m

```diff
@@ -92,7 +92,7 @@ - (NSString *)defaultAccessibilityLabelForDate:(NSDate *)date formatter:(NSDateF
     NSString *accessibilityLabel = [super defaultAccessibilityLabelForDate:date formatter:formatter];
 
     if (self.priceLabelCellData.price != nil) {
-        accessibilityLabel = [NSString stringWithFormat:@"%@ %@", accessibilityLabel, self.priceLabelCellData.price];
+        accessibilityLabel = [NSString stringWithFormat:@"%@, %@", accessibilityLabel, self.priceLabelCellData.price];
```

Comment:
The comma makes sure we don't "glue" the date to the price. 
If the price is something like 235 $, voiceOver would glue "May 5, 2023 235 $" and then spell it out as 2 0 2 3 2 3 5 dollar. Listing the numbers individually.